Added browse button to right click "move storage" option. Adds #7

Mattie432 opened the pull request:
This push adds a browse option to the right click, "move storage" button. It does this by adding a new menu item to the right click contect menu called "Move Storage Advanced" as this runs the same code as the normal move dialogue button but has an extra browse button included. Unfortunatly I could not remove the origional 'move storage' button from the right click menu so there are two buttons to move a files location (could be changed later).

Feature request from [#7](https://github.com/dredkin/deluge-rbb/issues/7)

![Image 1](http://i.imgur.com/Nkks5eo.png "Image 1")
![Image 2](http://i.imgur.com/yqGgXZ3.png "Image 2")

dredkin commented:
You know like they said: Good Ideas come to several minds simultaneously :)
I've just started to implement feature just like this.
I'll see your code!

Mattie432 commented:
Updated to no longer add an additional button to the right-click menu. The original "move" button now opens the dialog with the "browse" button. Removed the preferences pane which fixes #3. Cleaned up the debugs that were left in.
